pkg/cli/cliflags/flags.go

```diff
@@ -272,9 +272,29 @@ sessions.`,
 		Name:      "join",
 		Shorthand: "j",
 		Description: `
-The address of a node which acts as bootstrap when a new node is
-joining an existing cluster. This flag can be specified
-separately for each address, for example:
+The addresses for connecting a node to a cluster.
+<PRE>
+
+</PRE>
+When starting a multi-node cluster for the first time, set this flag
```

Comment:
Actually, for further clarity: `against any of the nodes` > `against one of the nodes`

Reply:
Done.